functional_tests.go

```diff
@@ -863,26 +1027,37 @@ func testRemovePartiallyUploaded() {
 	}()
 
 	objectName := bucketName + "-resumable"
+	args["objectName"] = objectName
+
 	_, err = c.PutObject(bucketName, objectName, reader, "application/octet-stream")
 	if err == nil {
-		logger().Fatal("Error: PutObject should fail.")
+		failureLog(function, args, startTime, "", "PutObject should fail", err).Fatal()
```

Comment:
`err` is nil. No point in logging it in.
`failureLog(function, args, startTime, "", "PutObject was supposed to fail"~~, err~~).Fatal()`